decidim-proposals/app/cells/decidim/proposals/proposal_m_cell.rb

```diff
@@ -94,6 +94,14 @@ def progress_bar_subtitle_text
       def can_be_followed?
         !model.withdrawn?
       end
+
+      def has_image?
+        model.attachments.first.present? && model.component.settings.allow_card_image
```

Comment:
What if the attachment is not an image file (eg. a PDF)? Can this happen?

Reply:
If the attachment is a non-image, the card is displayed like a standard card.